js/main.js

```diff
@@ -0,0 +1,33 @@
+var $image = document.querySelector('img');
+var $photoUrl = document.querySelector('.photo-url');
+var $formEntry = document.querySelector('form');
+var $titleInput = document.getElementsByName('title');
+var $photoInput = document.getElementsByName('photo-URL');
+var $notesInput = document.getElementsByName('notes');
+var formObj = {};
+
+window.addEventListener('beforeunload', handleBeforeUnload);
+function handleBeforeUnload(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('Journal Entry', dataJSON);
+}
```

Comment:
Please move this to `data.js` so that it is in the same file as the variable it is preserving.